watsong/spotify.py

```diff
@@ -52,7 +51,7 @@ def get_memo(name: str) -> Any:
 def set_memo(structure: Any, name: str) -> None:
     try:
         with open(f"{name}.pickle", "wb") as file:
-            pickle.dump(structure, file, protocol=pickle.HIGHEST_PROTOCOL)
```

Comment:
Same as above

Reply:
It was pickle5 before because I copy pasted the protocol=pickle.HIGHEST_PROTOCOL part, but if you think about it this can vary with versions of python which is why I went down to version 4 which works down to python 3.4

Reply:
Makes sense to me